README.md

```diff
@@ -559,6 +560,249 @@ The model can also be downloaded and served locally.
+
+### AVERAGE:
+8.07 (across 91)
+
+### TURN ONE:
+8.64
+
+### TURN TWO:
+7.19
+
+### ERROR RATE:
+0.43
+```
+
+Below is an example of running MTBench with local GGUF models:
+
+```bash
+$ export ILAB_MODELS_DIR=/home/example-user/.local/share/instructlab/models
+$ ilab model evaluate --benchmark mt_bench --model $ILAB_MODELS_DIR/granite-7b-lab-Q4_K_M.gguf --judge-model $ILAB_MODELS_DIR/granite-7b-lab-Q4_K_M.gguf
+...
+# SKILL EVALUATION REPORT
+
+## MODEL
+/home/example/.local/share/instructlab/models/granite-7b-lab-Q4_K_M.gguf
+
+### AVERAGE:
+5.0 (across 1)
+
+### TURN ONE:
+5.0
+
+### TURN TWO:
+N/A
+
+### ERROR RATE:
+0.99
+```
+
+> [!NOTE]
+> Currently, MTBench must be used with local models. Using models directly from Hugging Face without downloading them is unsupported.
+
+#### Running MTBenchBranch
+
+Below is an example of running MTBenchBranch with local safetensor models:
+
+```bash
+$ export ILAB_MODELS_DIR=/home/example-user/.local/share/instructlab/models
+$ export ILAB_TAXONOMY_DIR=/home/example-user/.local/share/instructlab/taxonomy
+$ ilab model evaluate --benchmark mt_bench_branch \
+   --model $ILAB_MODELS_DIR/instructlab/granite-7b-lab \
+   --judge-model $ILAB_MODELS_DIR/instructlab/granite-7b-lab \
+   --base-model $ILAB_MODELS_DIR/instructlab/granite-7b-lab \
+   --taxonomy-path $ILAB_TAXONOMY_DIR \
+   --branch rc \
+   --base-branch main
+...
+# SKILL EVALUATION REPORT
+
+## BASE MODEL
+/home/example/.local/share/instructlab/models/instructlab/granite-7b-lab
+
+## MODEL
+/home/example/.local/share/instructlab/models/instructlab/granite-7b-lab
+
+### IMPROVEMENTS:
+1. compositional_skills/extraction/receipt/markdown/qna.yaml (+4.0)
+2. compositional_skills/STEM/science/units_conversion/temperature_conversion/qna.yaml (+3.0)
+3. compositional_skills/extraction/commercial_lease_agreement/bullet_points/qna.yaml (+3.0)
+...
+
+### REGRESSIONS:
+1. compositional_skills/extraction/abstractive/title/qna.yaml (-5.0)
+2. compositional_skills/extraction/receipt/bullet_points/qna.yaml (-4.5)
+3. compositional_skills/writing/grounded/summarization/wiki_insights/one_line/qna.yaml (-4.0)
+...
+
+### NO CHANGE:
+1. compositional_skills/STEM/math/reasoning/qna.yaml
+2. compositional_skills/extraction/commercial_lease_agreement/csv/qna.yaml
+3. compositional_skills/roleplay/explain_like_i_am/graduate/qna.yaml
+...
+
+### NEW:
+1. compositional_skills/linguistics/organize_lists/qna.yaml
+2. compositional_skills/extraction/invoice/plain_text/qna.yaml
+3. compositional_skills/writing/grounded/summarization/wiki_insights/concise/qna.yaml
+...
+
+### ERROR RATE:
+0.32
+```
+
+Below is an example of running MTBench with local GGUF models:
+
+```bash
+ilab model evaluate --benchmark mt_bench_branch --model $ILAB_MODELS_DIR/granite-7b-lab-Q4_K_M.gguf --judge-model $ILAB_MODELS_DIR/granite-7b-lab-Q4_K_M.gguf --base-model $ILAB_MODELS_DIR/granite-7b-lab-Q4_K_M.gguf --taxonomy-path $ILAB_TAXONOMY_DIR --branch rc --base-branch main
```

Comment:
Should you define `$ILAB_MODELS_DIR` here? Alternatively, not to duplicate, you could put setting up variables into a common pre-requisite section above all the examples.